spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -636,6 +640,30 @@ def setup_arrow_buttons(self):
 
         return up_btn, down_btn
 
+    def setup_thumbnail_scrollbar_width(self):
+        """
+        Set the width of this scrollbar to that of the thumbnails.
+        """
+        # Create a fake thumbnail.
+        thumbnail = FigureThumbnail(parent=self)
+        thumbnail.setAttribute(Qt.WA_DontShowOnScreen, True)
+        thumbnail.canvas.setFixedSize(THUMBNAIL_SIZE, THUMBNAIL_SIZE)
+        thumbnail.show()
+
+        # Calculate and set the width of the thumbnail scrollbar using
+        # the size of the fake thumbnail.
+        thumbnail_scrollbar_width = (
+            thumbnail.width() +
+            self.scrollarea.viewportMargins().left() +
+            self.scrollarea.viewportMargins().right() +
+            2 * self.lineWidth())
+        if is_dark_interface():
+            # This is required to take into account some hard-coded padding
```

Comment:
```suggestion
            # TODO: This is required to take into account some hard-coded padding
```

Maybe to point that this could be improved in the future?

Reply:
> I have some comments regarding the widget but this can be addressed on a separate PR if you wish so. I write them here cause I noticed this while reviewing the PR
> 
>     1. The arrows look rather slim and are hard to get when using the mouse.
> 
> 
> <img alt="Screen Shot 2019-07-02 at 20 22 13" width="199" src="https://user-images.githubusercontent.com/3627835/60556482-21dd6480-9d07-11e9-9916-1f634194fe9a.png">
> 
>     1. The separator has the appearance of allowing the user to change the width gradually, but currently the behavior is more of something being toggled (a button instead of a draggable area)
> 
> 
> ![plots](https://user-images.githubusercontent.com/3627835/60556568-99ab8f00-9d07-11e9-935d-6781402304c3.gif)
> 
> I think we can improve that UX @jnsebgosselin, thoughts?

These are good point. Please open a new issue for these and I will address them in a subsequent PR.

Reply:
I agree with this, but imo the comment then needs to be reworded so that it explains what needs to be done, not why this exception was added here, otherwise adding a TODO is kind of futile.

Reply:
Gimme a day or too just to look at this again with a fresh eye.